### ISSUE TITLE
regression of test_handles_cyclical_objects in -0.9.1

### ISSUE THREAD

**Issue by idella** [jsonpickle/jsonpickle#124]
Under pythons 2.7  3.3  3.4 get;

```python
FAIL: test_handles_cyclical_objects (jsonpickle_test.PicklingProtocol2TestCase)
----------------------------------------------------------------------
Traceback (most recent call last):
  File "/mnt/gen2/TmpDir/portage/dev-python/jsonpickle-0.9.1/work/jsonpickle-0.9.1/tests/jsonpickle_test.py", line 1263, in test_handles_cyclical_objects
    self.assertEqual(PickleProtocol2Thing, decoded.args[0].args[0].__class__)
AssertionError: <class 'jsonpickle_test.PickleProtocol2Thing'> != <class 'jsonpickle.unpickler._Proxy'>

----------------------------------------------------------------------
Ran 222 tests in 0.379s

FAILED (failures=1, skipped=2)
```

For version -0.9.0.

Ran between 221 - 223 tests in 0.264s   OK

**Comment by davvid** [comment]
The tests have been fixed in v0.9.2

**Comment by idella** [comment]
great thanks @davvid
